scripts/exgdas_atmos_verfrad.sh

```diff
@@ -136,19 +139,13 @@ fi
 #####################################################################
 # Postprocessing
 
-err=0
+export err=0
 if [[ ${data_available} -ne 1 ]]; then
-   err=1
-elif [[ ${rc_angle} -ne 0 ]]; then
-   err=${rc_angle}
-elif [[ ${rc_bcoef} -ne 0 ]]; then
-   err=${rc_bcoef}
-elif [[ ${rc_bcor} -ne 0 ]]; then
-   err=${rc_bcor}
-elif [[ ${rc_time} -ne 0 ]]; then
-   err=${rc_time}
+   export err=1
 fi
 
+export err=$((err + rc_angle + rc_bcoef + rc_bcor + rc_time))
```

Comment:
How does `err_chk` handle `err` when it's 2, 3, 4, or 5 (assuming these additions are 1)?

Reply:
It will call `err_exit` if `err` is anything but `0` and print the error code in the `FATAL ERROR` message.

Reply:
Gotcha ok, so it's not about the number meaning anything other than being greater than 0. Noted. I haven't worked with `err_chk` much yet so this is a learning experience for me. :)